src/views/Home.vue

```diff
@@ -33,6 +38,16 @@ export default defineComponent({
   setup() {
     const store = useStore();
 
+    const productAddedMessage = ref(false);
```

Comment:
missing type

Reply:
productAddedMessage name is suggesting that there's some kind of message stored, not a flag. I'd change that name. We usually use `is` prefix before variable name to higlight that its a binary flag